gen/generate_test.go

```diff
@@ -100,3 +103,18 @@ func TestLoadSingleRootKeyConcurrent(t *testing.T) {
 	}
 	wg.Wait()
 }
+
+func TestGenesisRoundoff(t *testing.T) {
+	verbosity := strings.Builder{}
+	genesisData := DefaultGenesis
+	genesisData.NetworkName = "wat"
+	genesisData.ConsensusProtocol = protocol.ConsensusCurrentVersion // TODO: also check ConsensusFuture ?
```

Comment:
I don't think that adding a different `ConsensusProtocol` would make any difference in the way we setup the accounts, wouldn't it ?